frontend/src/pages/clusterSettings/ClusterSettings.tsx

```diff
@@ -175,242 +170,228 @@ const ClusterSettings: React.FC = () => {
-                    type="text"
-                    aria-label="Toleration key"
-                    value={notebookTolerationSettings.key}
-                    placeholder={DEFAULT_TOLERATION_VALUE}
-                    validated={
-                      notebookTolerationSettings.error ? ValidatedOptions.error : undefined
+            }
+          >
+            <Stack hasGutter>
+              <StackItem>
+                <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsSm' }}>
+                  <Radio
+                    id="culler-timeout-unlimited"
+                    data-id="culler-timeout-unlimited"
+                    label="Do not stop idle notebooks"
+                    isChecked={cullerTimeoutChecked === CULLER_TIMEOUT_UNLIMITED}
+                    name={CULLER_TIMEOUT_UNLIMITED}
+                    onChange={radioCheckedChange}
+                    value={CULLER_TIMEOUT_UNLIMITED}
+                  />
+                  <Radio
+                    id="culler-timeout-limited"
+                    data-id="culler-timeout-unlimited"
+                    label="Stop idle notebooks after"
+                    isChecked={cullerTimeoutChecked === CULLER_TIMEOUT_LIMITED}
+                    name={CULLER_TIMEOUT_LIMITED}
+                    onChange={radioCheckedChange}
+                    value={CULLER_TIMEOUT_LIMITED}
+                    body={
+                      <InputGroup>
+                        <TextInput
+                          id="hour-input"
+                          style={{ maxWidth: '60px' }}
+                          name="hour"
+                          data-id="hour-input"
+                          type="text"
+                          aria-label="Culler Timeout Hour Input"
+                          value={hour}
+                          isDisabled={cullerTimeoutChecked === CULLER_TIMEOUT_UNLIMITED}
+                          onChange={(value: string) => {
+                            let newValue =
+                              isNaN(Number(value)) || !Number.isInteger(Number(value))
+                                ? hour
+                                : Number(value);
+                            newValue =
+                              newValue > MAX_HOUR
+                                ? MAX_HOUR
+                                : newValue < MIN_HOUR
+                                ? MIN_HOUR
+                                : newValue;
+                            // if the hour is max, then the minute can only be set to 0
+                            if (newValue === MAX_HOUR && minute !== MIN_MINUTE) {
+                              setMinute(MIN_MINUTE);
+                            }
+                            setHour(newValue);
+                          }}
+                        />
+                        <InputGroupText variant={InputGroupTextVariant.plain}>hours</InputGroupText>
+                        <TextInput
```

Comment:
Reeeally weird issue in safari, tested in firefox and chrome and works great.

<img width="781" alt="Screenshot 2023-03-02 at 11 17 16" src="https://user-images.githubusercontent.com/16117276/222400297-53ac5fa2-ec16-4dab-87d2-b0da103c80c0.png">

Reply:
Hmmm, weird. But I found put `Radio` inside `FlexItem` could solve this problem, nice catch!